data/human/human missions.txt

```diff
@@ -5428,3 +5428,66 @@ mission "Deal Change 2"
 				die
 			label skip
 			`	After a few minutes, the pirates finish unloading the cargo, and Ci steps forward, holding out a credit chip. "<payment>, for making the right choice." Ci and the pirates return to their ships as you take off and head towards the main spaceport of <planet>.`
+
+
+
+government "Escort (Betraying)"
```

Comment:
Placing a government that's only used by a single mission in the main governments file seems unhelpful - it makes the governments file larger and thus less convenient to look through, and it also means you need to look in two separate files to find out all the necessary information about the mission, rather than only one.

Reply:
I agree with you but thats the current convention, and changing that should be in another PR